### ISSUE TITLE
Linting shows weird cython errors ?

### ISSUE THREAD

**Issue by zcattacz** [ktnrg45/vs-code-cython#4]
```
from cython.operator import preincrement as incr, dereference as dref, address as addr
         ^-----------------------------------------------------Import "cython.operator" could not be resolvedCython[reportMissingImports]
...
cdef class Snapshot
    cdef double          ts
    cdef vector[unsigned short]* cnlnum
                 ^--------------------------------------------  Expected expression Cython
    cdef vector[cnldata_t]       cnldat
...
cdef class BaseTable:
    def __init__(self, iobuf: BytesIO ):
         ...
        self._iobuf = iobuf
    cdef _load_cols(self):
        cdef:
            vector[string] type_symbols = [b"i", b"d", b"?", b"d", b"s"]
            int            vpos      = 0
            unsigned char[:] view    = self._iobuf.getbuffer()
                                     ^------------------------------------------Expected (  Cython
                                         ^--------------------------------------Expected function name after def, cdef, cpdef   Cython
            unsigned char[:] fieldDefBuf
            unsigned char  col_cnt = view[0]
            int            rec_sz  = 0
        ...
    cdef _load_rows(self):
        cdef:
            int            vpos      = self.cols.size()*COL_DEF_SZ + 3
                                     ^------------------------------------------Expected (  Cython
                                         ^--------------------------------------Expected function name after def, cdef, cpdef   Cython
            unsigned char[:] view    = self._iobuf.getbuffer()
            unsigned char[:] buf_row
        ...

```

**Comment by ktnrg45** [comment]
Thanks for reporting this. All of the issues should be resolved in 0.1.2; except for the cython import error.

I'll look into resolving this.

**Comment by ktnrg45** [comment]
Added typings for cython.operator. Issues should be resolved.
